docs/data-sources/dns_zones.md

```diff
@@ -0,0 +1,99 @@
+---
+subcategory: "Domain Name Service (DNS)"
+---
+
+# huaweicloud_dns_zones
+
+Use this data source to get the list of DNS zones.
+
+## Example Usage
+
+```hcl
+variable "zone_type" {}
+variable "enterprise_project_id" {}
+
+data "huaweicloud_dns_zones" "test" {
+  zone_type             = var.zone_type
+  enterprise_project_id = var.enterprise_project_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the data source.
+  If omitted, the provider-level region will be used.
+
+* `zone_type` - (Required, String) Specifies the zone type. The value can be **public** or **private**.
+
+* `tags` - (Optional, String) Specifies the resource tag. The format is as follows: key1,value1|key2,value2.
+  Multiple tags are separated by vertical bar (|). The key and value of each tag are separated by comma (,). The tags
+  are in `AND` relationship. Exact matching will work. If the value starts with an asterisk (*), fuzzy matching will
+  work for the string following the asterisk.
+
+* `status` - (Optional, String) Specifies the zone status. Valid values are as follows:
+  + **ACTIVE**: Normal.
+  + **ERROR**: Failed.
+  + **PENDING_CREATE**: Creating.
+  + **PENDING_UPDATE**: Updating.
+  + **PENDING_DELETE**: Deleting.
```

Comment:
I think we can delete the status starts with **PENDING**

Reply:
OK